app/src/main/java/com/example/myapplication/utils/Utils.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.myapplication.utils
+
+const val EMPTY_STRING : String = ""
+const val NULL_RESULT : Float = 0F
+const val PLUS : String = "+"
+const val SUBTRACT : String = "-"
+const val DIVIDE : String = "/"
+const val MULTIPLICATION : String = "x"
```

Comment:
according to the rest, this should "multiply"

Reply:
done